server/views/pages/component.njk

```diff
@@ -130,6 +130,39 @@
             No Team slack &#129301; - please provide one.
           {% endif %}
           </td>
+        <tr>
+        <th>Production Alerts Slack Channel</th>
+          <td>
+          {% if component.alerts_prod_slack_channel | length %}
+            <a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ component.alerts_prod_slack_channel }}">
+            <image src="/assets/images/slack_favicon_32.png" width="32" height="32"></image>
+            {% if component.alerts_prod_slack_channel| length %}
+            {{ component.alerts_prod_slack_channel }}
+            {% else %}
+            Link
+            {% endif %}
+            (opens slack)</a>
+          {% else %}
+            No Production Alerts Slack Channel &#129301; - please provide one.
+          {% endif %}
+          </td>
+        </tr>
+        <tr>
+        <th>Non-Production Alerts Slack Channel</th>
+          <td>
+          {% if component.alerts_nonprod_slack_channel | length %}
+            <a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ component.alerts_nonprod_slack_channel }}">
+            <image src="/assets/images/slack_favicon_32.png" width="32" height="32"></image>
+            {% if component.alerts_nonprod_slack_channel | length %}
+            {{ component.alerts_nonprod_slack_channel }}
+            {% else %}
+            Link
```

Comment:
What does `Link` mean?

Reply:
Surely this must exist for you to get to this line?

Reply:
You've done the same `if` statement inside itself?

Reply:
Resolved this